utils/stopwatch.go

```diff
@@ -66,3 +66,10 @@ func (w *Watch) UnPause() {
 	}
 	w.mu.Unlock()
 }
+
+func (w *Watch) IsPaused() (paused bool) {
+	w.mu.Lock()
```

Comment:
```suggestion
	w.mu.RLock()
```

Honestly I think the locks on this package aren't needed anymore as this should only be interacted with from a controller process...